src/helpers.php

```diff
@@ -57,3 +57,17 @@ function git_dir()
         return realpath($gitDir);
     }
 }
+
+if (! function_exists('get_hook_contents')) {
+    /**
+     * Return hook contents
+     *
+     * @param array|string $contents
+     *
+     * @return string
+     */
+    function get_hook_contents($contents)
+    {
+        return is_array($contents) ? implode(" && ", $contents) : $contents;
```

Comment:
I would still like the commands to be on multiple lines. let's use `" && \" . PHP_EOL` as the separator

Reply:
Right, .lock will be more readable in multiline